src/components/main-nav/main-nav.scss

```diff
@@ -1,3 +1,266 @@
-nav {
-  text-align: center;
+@import '../shared/variables.scss';
+@import '../shared/typography.scss';
+@import '../shared/base.scss';
+@import '../shared/mixins.scss';
+
+.header {
+  width: 100%;
+  padding: .5em 1em;
+  border-bottom: .2px solid rgba($black, .5);
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    @include from(1024px) {
```

Comment:
I suggest using a mixing that doesn't require to pass a number itself, but instead a name like `large`